src/ResourceManager/KeyVault/ChangeLog.md

```diff
@@ -24,6 +24,10 @@
 ## Version 3.3.0
 
 ## Version 3.2.1
+* New/updated Cmdlets to support soft-delete for KeyVault certificates
```

Comment:
@dragav please move this to under `Current Release`; the versions that are already listed are the notes from previous releases